tripcode_test.go

```diff
@@ -11,25 +11,33 @@ func TestGenerateSalt(t *testing.T) {
 		"!@#$%^&*()": "G.",
 		"f}E":        ".E",
 		"©":          "H.",
-		"訛":          "H.",
+		"訛":         "H.",
 		"'":          "H.",
 	}
 	for pass, expected := range cases {
 		salt := generateSalt(pass)
 		if expected != salt {
-			t.Error("Expected "+expected+", got", salt)
+			t.Error("Expected \"%s\", got \"%s\"", expected, salt)
 		}
 	}
 }
 
+func TestSecureTripcode(t *testing.T) {
+	expect := "PqG4A0fkUs"
+	trip := SecureTripcode("pass", "salt")
+	if trip != expect {
+		t.Errorf("SecureTripcode: expected \"%s\", got \"%s\"", expect, trip)
+	}
+}
+
 func TestTripcode(t *testing.T) {
 	cases := map[string]string{
 		"asd":        "TAPy3blMsc",
 		"adasd":      "IOuORdzMKw",
 		"!@#$%^&*()": "BpZUCmJAIQ",
 		"f}E":        "oUBoOTrysY",
-		"©":          "",
-		"訛":          "c8eDXvwFLQ",
+		"©":          "hEp4vbueZo", // Should be nothing?
```

Comment:
Good question. I checked several other implementations and they return `cPUZU5OGFs` as well, so you're right on this one.